src/containers/App/index.jsx

```diff
@@ -24,6 +26,7 @@ const App = () => (
       <Route path="/courses/:courseId" component={CoursewarePage} />
       <Route path="/faq" component={FAQSupportPage} />
       <Route path="/admin" component={CourseEnrollmentsPage} />
+      <Route path="/login" component={LoginPage} />
```

Comment:
We need to think about if this should live outside of `<App />` since the enterprise branding in the header/footer are here. Lets sync up on this to make sure we don't clobber this PR with the branding changes in https://github.com/edx/edx-portal/tree/feature/branding